cli/templatepush.go

```diff
@@ -100,6 +100,15 @@ func (r *RootCmd) templatePush() *serpent.Command {
 				return err
 			}
 
+			// If user hasn't provided new provisioner tags, inherit ones from the active template version.
+			if len(tags) == 0 && template.ActiveVersionID != uuid.Nil {
+				templateVersion, err := client.TemplateVersion(inv.Context(), template.ActiveVersionID)
+				if err != nil {
+					return err
+				}
+				tags = templateVersion.Job.Tags
+			}
```

Comment:
suggestion: log the re-used tags for transparency

Reply:
Could be sent to stout tom inform the user too.

Reply:
I decided to stick with `logger.Info()` 👍